docs/index.md

```diff
@@ -67,7 +67,6 @@ Installing the latest updown plugin will create a config file (`~/.steampipe/con
 connection "updown" {
   plugin     = "updown"
   api_key    = "9m_kAcfuTlW_JCrvoMYK6g"
-  api_secret = "lEEDVf3SgyQWckN3ASqMpXWpCixkwMzgnZY7"
```

Comment:
we can put this format `ro-bwABcDTDrMsnupyLopJ1` as we are saying Read-Only permission required only